bones/relationalBone.py

```diff
@@ -442,10 +442,22 @@ def fromClient( self, valuesCache, name, data ):
 			if self.using is not None:
 				refSkel = self._usingSkelCache
 				refSkel.setValuesCache({})
+				
 				if not refSkel.fromClient( r["reltmp"] ):
-					for k,v in refSkel.errors.items():
-						errorDict[ "%s.%s.%s" % (name,tmpList.index(r),k) ] = v
+					for k, v in refSkel.errors.items():
```

Comment:
You could do this purely client side, the necessary information (relationalBoneKey, Index, refSkelBoneKey) are present. But im not sure what's the better way; maybe we should check it on the next ViUR meeting